sickchill/views/manage/add_shows.py

```
@@ -87,28 +86,64 @@ def searchIndexersForShowName(self):
+            prev = deduped.get(key)
+            if prev is None or float(item.get("score") or 0) > float(prev.get("score") or 0):
+                deduped[key] = item
+        final_results = list(deduped.values())
 
         if exact in [True, "1"]:
-            logger.debug(_("Filtering and sorting out excess results because exact match was checked"))
-            final_results = [item for item in final_results if search_term.lower() in item[4].lower()]
-            final_results.sort(key=itemgetter(4))
-            final_results.sort(key=lambda x: x[4].lower().index(search_term.lower()))
-            final_results.sort(key=lambda x: x[4].lower() == search_term.lower(), reverse=True)
+            logger.debug(_("Filtering results because exact match was checked"))
```

Ruff (0.16.1)

[error] Undefined name `_`

(F821)